lib/capybara_select2/helpers.rb

```diff
@@ -26,6 +26,7 @@ def select2(*args)
 
         find_options = { text: value }
         find_options[:match] = options[:match] if options[:match]
+        find_options[:exact_text] = options[:exact_text] if options[:exact_text]
```

Comment:
how about

```ruby
find_options = options.slice(:match, :exact_text).merge({ text: value })
```
?

Reply:
Absolutely! That makes it look much cleaner.